src/trove_classifiers/__init__.py

```diff
@@ -280,6 +280,7 @@
     "License :: OSI Approved :: Common Public License",
     "License :: OSI Approved :: Eclipse Public License 1.0 (EPL-1.0)",
     "License :: OSI Approved :: Eclipse Public License 2.0 (EPL-2.0)",
+    "License :: OSI Approved :: Educational Community License v2.0 (ECL-2.0)",
```

Comment:
```suggestion
    "License :: OSI Approved :: Educational Community License, Version 2.0 (ECL-2.0)",
```
A small nit, but we should try to keep this as close as possible to the OSI name for the license.